### PR TITLE
Fix toggle styling

### PR DESCRIPTION
**Pull request by BalbinaK**
## Description
- update Warp dependencies to include the latest changes
- indicate that `small` prop in Toggle will only work with `radio-button` type
- add `small` radio-button Toggle example
- list `indeterminate`, `disabled` and `invalid` props in Vue Toggle
- list `disabled` prop in React Toggle

Screenshot of checkbox examples in the tech-docs
Screenshot of radio examples in the tech-docs
Screenshot of radiobuttons examples in the tech-docs

### CODE REVIEW

File: docs/api-table.js [hunk 1]
```diff
@@ -779,11 +779,11 @@ export const react = {
       ['noVisibleLabel', 'boolean', '', 'Whether label should be invisible'],
       ['checked', 'boolean', '', 'Whether the single option should be checked (controlled)'],
       ['defaultChecked', 'boolean', '', 'Whether the single option should be checked on mount (uncontrolled)'],
-      ['small', 'boolean', '', 'Whether the elements should be small'],
-      ['optional', 'boolean', '', 'Whether the toggle is optional Appends (valgfritt) to the end of the title for indication'],
-      ['equalWidth', 'boolean', '', 'Will make each radio-button equal width'],
       ['indeterminate', 'boolean', '', 'Whether a single option is indeterminate, or "partially checked." The checkbox will appear with a small dash instead of a tick to indicate that the option is not exactly checked or unchecked.'],
+      ['optional', 'boolean', '', 'Whether the toggle is optional Appends (valgfritt) to the end of the title for indication'],
```
Comment (felicia-haggqvist) [inline docs/api-table.js]: Should the Norwegian word `valgfritt` be here? Shouldn't it be in english as default? :)

Comment (felicia-haggqvist) [reply docs/api-table.js]: Oh I just realized that you had just moved this line and that it wasn't your change, so maybe we should save my previous question for another ticket? 😄

Comment (BalbinaK) [reply docs/api-table.js]: Still, good point - missed that :D Fixed in [0854c34](https://github.com/warp-ds/tech-docs/pull/189/commits/0854c3419d9769f2e8702ab51d3f761924d64e06)

**Comment by BalbinaK** [comment]
In order to keep things in the same context, I added a blog post on recent Warp releases to the What's new section in [8dc54d2](https://github.com/warp-ds/tech-docs/pull/189/commits/8dc54d221d9674a702fb566f155c3d02355a4e25)

**Review by felicia-haggqvist — APPROVED** [review]
LGTM! :)

**Pull request merged**